cordapp-upgrades/contracts/v1-contract/src/main/resources/META-INF/services/net.corda.webserver.services.WebServerPluginRegistry

```diff
@@ -0,0 +1,2 @@
+# Register here any CorDapp plugins extending from net.corda.webserver.services.WebServerPluginRegistry.
+net.corda.examples.obligation.ObligationPlugin
```

Comment:
I imagine the original has a web frontend, but do we need it for our purposes here? Getting rid of the web code would reduce the size of this PR quite a bit I think.